config/config.go

```diff
@@ -0,0 +1,121 @@
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/mitchellh/go-homedir"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	// Filename into which Config is serialized
+	Filename = "config.yaml"
+
+	// DefaultConfigPath is used in the unlikely event that
+	// the user has no home directory (no ~), there is no
+	// XDG_CONFIG_HOME set
+	DefaultConfigPath = ".config/func"
+
+	// DefaultLanguage is intentionaly undefined.
+	DefaultLanguage = ""
+)
+
+type Config struct {
+	// Language Runtime
+	Language string `yaml:"language"`
+
+	// Confirm Prompts
+	Confirm bool `yaml:"confirm"`
```

Comment:
@lkingland so the idea here is to add for example a global default registry for all functions? I mean in another PR

Reply:
Yes, along with any other settings who would benefit from a shared configurable default.

Reply:
💯